webknossos/examples/learned_segmenter.py

```diff
@@ -20,7 +20,7 @@ def main() -> None:
 
     # Step 1: Download the dataset and our training data annotation from webKnossos to our local computer
     training_data_bbox = wk.BoundingBox.from_tuple6(
-        annotation.skeleton.user_bounding_boxes[0]
+        annotation.skeleton.user_bounding_boxes[0]  # type: ignore[index]
```

Comment:
Thanks for pushing this. Yesterday, I was unsure how to fix this (strictly speaking one would need to check that `user_bounding_boxes` is not None). Ideally, it would always default to an empty list. However, that's likely out-of-scope for this PR. So, let's tackle it at a different point.

Reply:
I agree, cleaning things up a bit more in the skeleton class is definitely a good idea, but let's defer this.